packages/vats/src/provisionPool.js

```diff
@@ -15,16 +17,33 @@ const { details: X, quote: q } = assert;
 
 const privateArgsShape = harden({
   poolBank: M.eref(M.remotable('bank')),
+  initialPoserInvitation: M.remotable('Invitation'),
+  storageNode: M.eref(M.remotable('storageNode')),
+  marshaller: M.eref(M.remotable('marshaller')),
 });
 
+/**
+ * @typedef {object} MetricsNotification
+ * Metrics naming scheme is that nouns are present values and past-participles
+ * are accumulative.
+ *
+ * @property {bigint} provisionedCount  count of new wallets provisioned
```

Comment:
this is an accumulative value. I think it should be `walletsProvisioned`. 

requesting change for this.

Reply:
ok. changed.